.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```

Comment:
Whhaaat? my comment disappeared.

Anyways it was about this desktp and smartphone fields. Since this is a backend library we should offer some examples that align better. Such as python version, base open release, containerization mechanism or other plugins installed at the same time. This could even be installed as a dependency in something other than edx-platform so that is another option.

Reply:
True! I just used the default ones. I'll customize them :)

Reply:
@felipemontoya I decided to use the defaults from the library. So these changes are not needed anymore.